README.md

```diff
@@ -89,6 +89,12 @@ $ fil-profile run yourscript.py --input-file=yourfile
 
 And it will generate a report.
 
+As of version 0.11, you can also run it like this:
+
+```
+$ python -m filprofiler run yourscript.py --input-file=yourfile
```

Comment:
@itamarst thanks you very much for the change, a small question can the default be without the run word?

Reply:
Sadly no, because then it'll be hard to add additional commands. For example, I want things like `fil-profile run-again`, which runs the last command again and then shows you the difference in memory usage.